src/output/plugins/PipeOutputPlugin.cxx

```diff
@@ -84,12 +97,30 @@ PipeOutput::Create(const ConfigBlock &block, Error &error)
 	return po;
 }
 
+#define set_MPDPIPE_vars(audio_format) {\
+	char strbuf[8];\
+	setenv("MPDPIPE_BITS", sample_format_to_string(audio_format.format), 1);\
+	snprintf(strbuf, sizeof(strbuf), "%u", audio_format.sample_rate);\
+	setenv("MPDPIPE_RATE", strbuf, 1);\
+	snprintf(strbuf, sizeof(strbuf), "%u", audio_format.channels);\
+	setenv("MPDPIPE_CHANNELS", strbuf, 1);\
+}
+
 inline bool
 PipeOutput::Open(AudioFormat &audio_format, Error &error)
 {
+#ifdef WIN32
+	set_MPDPIPE_vars(audio_format);
+	fh = popen(cmd.c_str(), "w");
```

Comment:
This commit re-adds code that has been removed before. Since I don't merge your original (bad) commit, I can't merge this one, because it conflicts.